torchtext/experimental/vectors.py

```diff
@@ -0,0 +1,211 @@
+import csv
+import io
+import os
+
+import torch
+from torch import Tensor
+import torch.nn as nn
+
+from torchtext.utils import (
+    download_from_url,
+    extract_archive
+)
+
+
+def fast_text(language="en", unk_tensor=None):
```

Comment:
Can we consolidate `fast_text` and `glo_ve ` func? Seems like several parts are similar.

Reply:
Yup I was thinking about this. Do you think I should create more utility functions? And if so I was also wondering about the best place to put them. I.e. should I keep them in the `vectors.py` file or create a new file under `experimental/utils.py`?

Reply:
for now, you can create some private funcs in vectors file here. No need to push them to utils unless they are very generic.